core-bundle/contao/templates/modules/mod_breadcrumb.html5

```diff
@@ -1,6 +1,17 @@
+<?php
+
+$this->wrapperAttributes = $this
+    ->attr($this->cssID)
+    ->addClass([$this->class, 'block'])
+    ->addStyle($this->style)
+    ->set('aria-label', 'Breadcrumb')
+    ->mergeWith($this->wrapperAttributes)
+;
+
+?>
 
 <!-- indexer::stop -->
```

Comment:
@fritzmg Why did you not extend from `block_unsearchable` here?

Reply:
Because this templates uses `<nav>`, not `<div>`.